src/api/db/models/Image.js

```diff
@@ -439,13 +442,14 @@ export class ImageModel {
         }
         console.log('ImageModel.updateObjects - operations: ', JSON.stringify(operations));
         return await Image.bulkWrite(operations);
-
       }, { retries: 2 });
     };
 
     try {
       const res = await operation(input);
       console.log('ImageModel.updateObjects - Image.bulkWrite() res: ', JSON.stringify(res.getRawResponse()));
+      const imageIds = [...new Set(input.updates.map((update) => update.imageId))];
```

Comment:
Get imageIds into an array, turn it into a set to remove duplicates, and then back to array so we can iterate through it.